docs/security/cve/zipbombs-and-octopus-deploy.md

```diff
@@ -0,0 +1,59 @@
+---
+title: ZipBombs and Octopus Deploy
+description: Octopus Server actively prevents ZipBomb-based denial-of-service attacks using a number of preventative measures.
+position: 1
```

Comment:
Unrelated, but what's the `position 1` about in this table? I'm missing the broader context clearly, so just thought I'd ask.

Reply:
Something about how the doc rendering engine organises related pages. All the other sibling CVE pages have the same thing, so I just copy-pasted.